PSScriptAnalyzerSettings.psd1

```diff
@@ -0,0 +1,67 @@
+@{
```

Comment:
These rules control the linter.  If you disagree with any of these, feel free to change it here.  Make sure to check it in so everyone is consistent.